components/org.wso2.carbon.identity.recovery/src/main/java/org/wso2/carbon/identity/recovery/handler/UserEmailVerificationHandler.java

```diff
@@ -129,6 +132,10 @@ public void handleEvent(Event event) throws IdentityEventException {
                 //Not required to handle in this handler
             } else if (IdentityRecoveryConstants.VERIFY_EMAIL_CLIAM.equals(claim.getClaimUri())) {
                 if (isNotificationInternallyManage) {
+
+                    if (log.isDebugEnabled()) {
+                        log.debug("Email verified notification is sending to user: " + user.getTenantDomain());
```

Comment:
Email verified notification is sending to user --> Sending email verification to user